app/views/container_topology/show.html.haml

```diff
@@ -24,56 +25,56 @@
   .legend
     %label#selected
     %div{'ng-if' => "kinds"}
-      %kubernetes-topology-icon{:kind => "ContainerReplicator", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+      %kubernetes-topology-icon{tooltipOptions, :kind => "ContainerReplicator"}
```

Comment:
@abonas yup, it does seem to work, but ..looks suprising, because it's not a valid ruby hash literal (and indeed, works only in HAML and only if you do `tooltipOptions` first).

OTOH, `hash = { :foo => bar, **hash2 }` is valid ruby to expand the keys and values from `hash2` into `hash`. (I'ts like `[ 1, 2, *array ]` but for hashes.)